gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -47,6 +47,10 @@
     <module name="BeforeExecutionExclusionFileFilter">
         <property name="fileNamePattern" value="module\-info\.java$"/>
     </module>
+    <module name="LineLength"> <!-- Java Style Guide: No line-wrapping -->
```

Comment:
This was mentioned in the changes it's just hard to decipher things from their long list of changes